src/metabolic_ninja/tasks.py

```diff
@@ -379,3 +402,32 @@ def persist(result, job_id):
         session.add(job)
         session.commit()
     return result
+
+
+@celery_app.task()
+def notify(result, job_id, product_name, organism_id, user_name, user_email,
+           organism_name):
+    try:
+        sendgrid = SendGridAPIClient()
+        mail = Mail()
+        mail.from_email = Email("DD-DeCaF <hello@dd-decaf.eu>")
```

Comment:
How about a slightly different e-mail address?
```suggestion
        mail.from_email = Email("DD-DeCaF <notifications@dd-decaf.eu>")
```